site/content/docs/4.6/components/navs.md

```diff
@@ -396,9 +396,9 @@ To help fit your needs, this works with `<ul>`-based markup, as shown above, or
 <div class="bd-example bd-example-tabs">
```

Comment:
(out from this PR) This example is broken using keyboard -> Not accessible and not the only one

Reply:
Tackled by #1294